NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2019 Composewell Technologies
```

Comment:
See https://github.com/composewell/unicode-data/blob/master/NOTICE .  It should be the same here.